p2p_wallet/Scenes/Sell/SellCoordinator.swift

```diff
@@ -13,9 +14,31 @@ final class SellCoordinator: Coordinator<SellCoordinatorResult> {
     }
 
     override func start() -> AnyPublisher<SellCoordinatorResult, Never> {
+        // create SellViewModel
         let viewModel = SellViewModel()
+        
+        // scene navigation
+        viewModel.navigationPublisher
+            .sink { [unowned self] scene in
+                switch scene {
+                case .moonpayWebpage(let url):
+                    navigateToMoonpay(url: url)
+                default:
+                    break
+                }
+            }
+            .store(in: &subscriptions)
+        
+        // create viewController
         let vc = UIHostingController(rootView: SellView(viewModel: viewModel))
         navigationController.pushViewController(vc, animated: true)
         return vc.deallocatedPublisher()
     }
+    
+    // MARK: - Navigation
+    private func navigateToMoonpay(url: URL) {
```

Comment:
should we make a coordinator, so it will close the VC when the right redirect is detected?